akka-grpc-play-quickstart-scala/project/plugins.sbt

```diff
@@ -10,5 +10,5 @@ addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0-M3")
 addSbtPlugin("com.typesafe.sbt" % "sbt-play-enhancer" % "1.2.2")
 
 // Akka GRPC
-addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.4.1")
+addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.5.0")
```

Comment:
Does this work? I thought `addSbtPlugin("com.lightbend.akka.grpc" %% "sbt-akka-grpc" % "0.5.0")` included an issue on the play generator.

Reply:
I have done so before and it works. Unless there is a new bug introduced in the meantime.

Reply:
I forgot, the bug is only on `javadsl` (IIRC in the generated Guice `Module`)